### ISSUE TITLE
Python 3.8 gives a warning on string comparisons

### ISSUE THREAD

**Issue by giadarol** [PyCOMPLETE/PyHEADTAIL#116]
```
/afs/cern.ch/work/g/giadarol/sim_workspace_mpi_py3/PyHEADTAIL/PyHEADTAIL/cobra_functions/pdf_integrators_2d.py:56: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if direction is 'x':
/afs/cern.ch/work/g/giadarol/sim_workspace_mpi_py3/PyHEADTAIL/PyHEADTAIL/cobra_functions/pdf_integrators_2d.py:58: SyntaxWarning: "is" with a literal. Did you mean "=="?
  elif direction is 'y':
/afs/cern.ch/work/g/giadarol/sim_workspace_mpi_py3/PyHEADTAIL/PyHEADTAIL/cobra_functions/pdf_integrators_2d.py:86: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if direction is 'x':
/afs/cern.ch/work/g/giadarol/sim_workspace_mpi_py3/PyHEADTAIL/PyHEADTAIL/cobra_functions/pdf_integrators_2d.py:88: SyntaxWarning: "is" with a literal. Did you mean "=="?
  elif direction is 'y':
/afs/cern.ch/work/g/giadarol/sim_workspace_mpi_py3/PyHEADTAIL/PyHEADTAIL/cobra_functions/pdf_integrators_2d.py:177: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if direction is 'x':
/afs/cern.ch/work/g/giadarol/sim_workspace_mpi_py3/PyHEADTAIL/PyHEADTAIL/cobra_functions/pdf_integrators_2d.py:179: SyntaxWarning: "is" with a literal. Did you mean "=="?
  elif direction is 'y':
/afs/cern.ch/work/g/giadarol/sim_workspace_mpi_py3/PyHEADTAIL/PyHEADTAIL/cobra_functions/pdf_integrators_2d.py:219: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if direction is 'x':
/afs/cern.ch/work/g/giadarol/sim_workspace_mpi_py3/PyHEADTAIL/PyHEADTAIL/cobra_functions/pdf_integrators_2d.py:221: SyntaxWarning: "is" with a literal. Did you mean "=="?
  elif direction is 'y':
```

**Comment by giadarol** [comment]
Some fixes (the ones I could find) are in PR #117

**Comment by aoeftiger** [comment]
I think there were more of this type which you collected in https://github.com/PyCOMPLETE/PyHEADTAIL/issues/108 as well. Can probably close both issues with #117 right?

**Comment by lmether** [comment]
Fixed in [v1.16.0](https://github.com/PyCOMPLETE/PyHEADTAIL/releases/tag/v1.16.0)
